Sources/Identity.swift

```diff
@@ -62,6 +65,55 @@ import Foundation
     /// Handles events requesting for identifiers. Dispatches response event containing the identifiers. Called by listener registered with event hub.
     /// - Parameter event: the identity request event
     private func handleIdentityRequest(event: Event) {
+        if event.urlVariables {
+            processGetUrlVariablesRequest(event: event)
+        } else {
+            processGetIdentifiersRequest(event: event)
+        }
+    }
+
+    /// Handles events requesting for url variables. Dispatches response event containing the url variables string.
+    /// - Parameter event: the identity request event
+    func processGetUrlVariablesRequest(event: Event) {
+        let emptyResponseEvent = event.createResponseEvent(name: IdentityConstants.EventNames.IDENTITY_RESPONSE_URL_VARIABLES,
+                                                           type: EventType.edgeIdentity,
+                                                           source: EventSource.responseIdentity,
+                                                           data: [IdentityConstants.EventDataKeys.URL_VARIABLES: ""])
+
+        guard let configurationSharedState = getSharedState(extensionName: IdentityConstants.SharedState.Configuration.SHARED_OWNER_NAME, event: event)?.value else {
```

Comment:
If updating `readyForEvent` to get the `.lastSet` Configuration state, also update here to get the `.lastSet`.